examples/case_studies/probabilistic_matrix_factorization.ipynb

```diff
@@ -16,22 +16,34 @@
      "name": "stdout",
```

Comment:
<p>I would use something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">csv_kwargs = dict(sep="\t", names=["userid", "itemid", "rating", "timestamp"])
try:
    data = pm.read_csv("../data/ml_100k_u.data", **csv_kwargs)
...
</pre><p> </p><p>To try and make the cell easier to read and pleasant to the view. Otherwise between the kwargs and black it ends up taking a lot of vertical space without being easier to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
yeah you are right, that's a good suggestion. thanks!